apps/authorizer/test/support/data_case.ex

```diff
@@ -13,4 +13,18 @@ defmodule Authorizer.DataCase do
       setup :verify_on_exit!
     end
   end
+
+  @doc """
+  A helper that transform changeset errors to a map of messages.
+      assert {:error, changeset} = Accounts.create_user(%{password: "short"})
+      assert "password is too short" in errors_on(changeset).password
+      assert %{password: ["password is too short"]} = errors_on(changeset)
+  """
+  def errors_on(changeset) do
+    Ecto.Changeset.traverse_errors(changeset, fn {message, opts} ->
+      Enum.reduce(opts, message, fn {key, value}, acc ->
+        String.replace(acc, "%{#{key}}", to_string(value))
+      end)
+    end)
```

Comment:
dont you think be more simple create a other project in umbrella that centralize helpers like that? @lcpojr 
I don't really like creating repeated funcs that way

Reply:
Actually that was my fault. We could use here the one from `resource_manager` because these applications depends on it.